src/main/java/racingcar/controller/RacingController.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.controller;
+
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.domain.Racing;
+import racingcar.domain.RacingResult;
+import racingcar.view.InputView;
+import racingcar.view.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingController {
+
+    public void run() {
+        String[] names = InputView.getCarNames();
+        int roundNum = InputView.getRoundNumber();
+
+        List<Car> carList = new ArrayList<>();
+        for (int i = 0; i < names.length; i++) {
+            carList.add(new Car(names[i]));
+        }
+
+        Cars cars = new Cars(carList);
```

Comment:
Nit: 자동차 이름 목록으로 `Cars` 인스턴스를 생성하는 정적 팩토리 메서드를 추가해봐도 좋을 것 같습니다.
```suggestion
        Cars cars = Cars.from(names);
```